src/__tests__/profiling-test.js

```diff
@@ -0,0 +1,266 @@
+// @flow
+
+describe('profiling', () => {
+  let React;
+  let ReactDOM;
+  let Scheduler;
+  let SchedulerTracing;
+  let TestRenderer;
+  let store;
```

Comment:
Off-topic but in https://github.com/bvaughn/react-devtools-experimental/pull/248 I found helpful for writing tests to annotate our globals with typehints (like `: Store`, `: Bridge`). What do you think?

Reply:
I'm not used to Flow-typing tests because the React repo (and most of my OSS projects) don't do it, but that sounds like a good idea.

Reply:
I think after 705cd9b, this new test should be pretty covered. Thoughts?

Reply:
Yeah looks great, you even got a type mismatch spotted :-)
```diff
-   return ids[0];
+   return parseInt(ids[0], 10);
```